.github/workflows/test-wordpress-version.yml

```diff
@@ -0,0 +1,61 @@
+name: 'Test WordPress Version'
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  wp4721:
```

Comment:
We want to test WP 4 ?

Reply:
We are not testing WP 4, but rather that the core version passed in the config installs the correct WP version. Since `@wordpress/env` allows installing any version, we ensure here that the version passed in the Action config is the one actually installed.